### PR TITLE
[topology] Added cron to run save_snapshot command

### PR DESCRIPTION
**Pull request by rohithasrk**
References #22

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -44,3 +44,7 @@ openwisp2_topology_update_frequency:
     day: "*"
     hour: "*"
     minute: "*/5"
+openwisp2_topology_save_snapshot_frequency:
+    day: "*/1"
```
Comment (nemesisdesign) [inline defaults/main.yml]: every day can be shortened to simply `*`

File: defaults/main.yml [hunk 2]
```diff
@@ -44,3 +44,7 @@ openwisp2_topology_update_frequency:
...
+    hour: "0"
+    minute: "0"
```
Comment (nemesisdesign) [inline defaults/main.yml]: let's start this process at 23:30 by default, that way we will capture the state of the network during the last hours of the day

**Comment by nemesisdesign** [comment]
@rohithasrk could you rebase your branch on the current master branch so the final merge with rebase strategy will result cleaner?

**Comment by rohithasrk** [comment]
Check now @nemesisdesign

**Pull request merged**